src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Logging/ProjectOutputWindowProjectLogger.cs

```diff
@@ -44,9 +42,9 @@ public void WriteLine(in StringFormat format)
                 // as via build), then we'll need to implement a better queueing mechanism.
                 _threadingService.RunAndForget(async () =>
                 {
-                    IVsOutputWindowPane pane = await _outputWindowProvider.GetOutputWindowPaneAsync();
+                    IVsOutputWindowPane? pane = await _outputWindowProvider.GetOutputWindowPaneAsync();
```

Comment:
This looks wrong - this should return non-null, I think?

Reply:
`ProjectOutputWindowPaneProvider.CreateOutputWindowPaneAsync` can return `null` (in command line builds), which surfaces here.